caffe2/sgd/adadelta_op_gpu.cu

```diff
@@ -0,0 +1,168 @@
+#include "adadelta_op.h"
+#include "caffe2/core/common_gpu.h"
+#include "caffe2/core/context_gpu.h"
+#include "caffe2/utils/mixed_utils.h"
+
+namespace caffe2 {
+
+__global__ void AdadeltaUpdate(
+    int N,
+    const float* w,
+    const float* g,
+    const float* h,
+    const float* d,
+    float* nw,
+    float* nh,
+    float* nd,
+    float epsilon,
+    float decay,
+    const float* lr) {
```

Comment:
Please make the output parameters at the end of the parameter list.
https://google.github.io/styleguide/cppguide.html#Output_Parameters